contracts/sol/MACI.sol

```diff
@@ -263,7 +263,9 @@ contract MACI is DomainObjs, ComputeRoot, MACIParameters, VerifyTally {
     function signUp(
         PubKey memory _userPubKey,
         bytes memory _signUpGatekeeperData,
-        bytes memory _initialVoiceCreditProxyData
+        bytes memory _initialVoiceCreditProxyData,
+        Message memory _message,
+        PubKey memory _encPubKey
```

Comment:
The user may want to vote for multiple options, so if we allow them to publish a message using `signUp` method, it would make sense to accept an array of messages here.

On the other hand, it is not necessary for `signUp` to accept messages at all, because an external contract can be created which allows to do `signUp` and make multiple `publishMessage` calls in one transaction (this is the approach I would prefer).

Reply:
> On the other hand, it is not necessary for signUp to accept messages at all, because an external contract can be created which allows to do signUp and make multiple publishMessage calls in one transaction (this is the approach I would prefer).

Perfect, this will make it easier to implement! I will revert this commit.